README.rst

```diff
@@ -1,207 +1,65 @@
-        wget -qO- https://storage.googleapis.com/symbiflow-arch-defs/artifacts/prod/foss-fpga-tools/symbiflow-arch-defs/continuous/install/112/20201208-080919/symbiflow-arch-defs-install-7c1267b7.tar.xz | tar -xJC $INSTALL_DIR/xc7/install
-        wget -qO- https://storage.googleapis.com/symbiflow-arch-defs/artifacts/prod/foss-fpga-tools/symbiflow-arch-defs/continuous/install/112/20201208-080919/symbiflow-arch-defs-xc7a50t_test-7c1267b7.tar.xz | tar -xJC $INSTALL_DIR/xc7/install
-        wget -qO- https://storage.googleapis.com/symbiflow-arch-defs/artifacts/prod/foss-fpga-tools/symbiflow-arch-defs/continuous/install/112/20201208-080919/symbiflow-arch-defs-xc7a100t_test-7c1267b7.tar.xz | tar -xJC $INSTALL_DIR/xc7/install
-        wget -qO- https://storage.googleapis.com/symbiflow-arch-defs/artifacts/prod/foss-fpga-tools/symbiflow-arch-defs/continuous/install/112/20201208-080919/symbiflow-arch-defs-xc7z010_test-7c1267b7.tar.xz | tar -xJC $INSTALL_DIR/xc7/install
-        conda deactivate
-
-* For the EOS S3 devices:
-
-.. code:: bash
-        :name: eos-s3-setup-toolchain
-
-        bash conda_installer.sh -b -p $INSTALL_DIR/eos-s3/conda
-        source "$INSTALL_DIR/eos-s3/conda/etc/profile.d/conda.sh"
-        conda env create -f eos-s3/environment.yml
-        conda activate eos-s3
-        wget -qO- https://quicklogic-my.sharepoint.com/:u:/p/kkumar/EWuqtXJmalROpI2L5XeewMIBRYVCY8H4yc10nlli-Xq79g?download=1 | tar -xJ -C $INSTALL_DIR/eos-s3/
-        conda deactivate
-
-Build Example Designs
----------------------
-
-With the toolchain installed, you can build the example designs.
-The example designs are provided in separate directories:
-
-* ``xc7`` directory for the Artix-7 devices
-* ``eos-s3`` directory for the EOS S3 devices
-
-
-
-Example designs for the Artix-7 devices:
-****************************************
-
-Before building any example, set the installation directory to match what you set it to earlier,
-
-.. code:: bash
-
-    export INSTALL_DIR=~/opt/symbiflow
-
-and prepare the environment:
-
-.. code:: bash
-        :name: xc7-prepare-env
-
-        # adding symbiflow toolchain binaries to PATH
-        export PATH="$INSTALL_DIR/xc7/install/bin:$PATH"
-        source "$INSTALL_DIR/xc7/conda/etc/profile.d/conda.sh"
-        conda activate xc7
-
-To build the counter example, run any or all of the following commands:
-
-.. code:: bash
-        :name: xc7-counter
-
-        pushd xc7/counter_test && TARGET="arty_35" make && popd
-        pushd xc7/counter_test && TARGET="arty_100" make && popd
-        pushd xc7/counter_test && TARGET="basys3" make && popd
-
-To build the picosoc example, run the following commands:
-
-.. code:: bash
-        :name: xc7-picosoc
-
-        pushd xc7/picosoc_demo && make && popd
-
-To build the litex example, run the following commands:
-
-.. code:: bash
-        :name: xc7-litex
+Please refer to the documentation in `docs <./docs>`_ for a proper guide on how
```

Comment:
I would instead link to the rendered readthedocs output at `symbiflow-examples.rtfd.io`.

Reply:
fixed